src/status_im/ui/screens/accounts/login/views.cljs

```diff
@@ -64,7 +64,7 @@
      [status-bar/status-bar]
      [login-toolbar can-navigate-back?]
      [components.common/separator]
-     [react/view styles/login-view
+     [react/scroll-view styles/login-view
```

Comment:
`:content-container-style` should be used i think

Reply:
seems like they both work because if I simply remove it the margin is gone